pkg/testutils/sensors/load.go

```diff
@@ -55,6 +55,63 @@ func findProgram(cache []*prog, name string, typ ebpf.ProgramType, t *testing.T)
 	return nil
 }
 
+func mergeSensorMaps(maps1, maps2 []SensorMap, progs1, progs2 []SensorProg, t *testing.T) ([]SensorMap, []SensorProg) {
```

Comment:
I guess this function is only intended for  testing?

If so, let's `t *testing.T`  place as the first argument.
Otherwise, let's avoid `t.Fatalf()` and have it return an error.

Reply:
yes, it's just for testing to merge tests maps, I'll move the 't' arg to the first position